### ISSUE TITLE
collaberate - Missing fieldset

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#122]
WAVE is alerting to the below on this page https://rrchnm.org/collaborate/ missing a fieldset.

![Screen Shot 2020-10-20 at 12 00 42 PM](https://user-images.githubusercontent.com/618307/96612597-eedc2d00-12cb-11eb-942e-a0e586e3840d.png)
![Screen Shot 2020-10-20 at 12 02 06 PM](https://user-images.githubusercontent.com/618307/96612716-18955400-12cc-11eb-872f-425606c57d12.png)

**Comment by kimisgold** [comment]
Fieldset has been applied and styled, ready for another look.
